Chef/Cookbooks/Relativity/attributes/default.rb

```diff
@@ -12,23 +12,23 @@
 default['file']['result']['destination_folder'] = 'C:\\vagrant'
 
 default['software']['jungum']['file_name'] = 'JungUm.msi'
-default['software']['jungum']['source'] = "\\\\kcura.corp\\shares\\Development\\DevEx\\DevVm\\Production\\DevVm_Install_Files\\Jungum\\#{default['software']['jungum']['file_name']}"
+default['software']['jungum']['source'] = "\\\\kcura.corp\\shares\\Development\\DevEx\\DevVm\\Development\\Aaron\\DevVm_Install_Files\\Jungum\\#{default['software']['jungum']['file_name']}"

```

Comment:
Should this file path be set to Production?

Reply:
@jake-martin They should updated now